src/BSeries.jl

```diff
@@ -1447,4 +1448,437 @@ include("latexify.jl")
     include("precompile.jl")
 end
 
+
+"""
+        is_energy_preserving(A,b,s)
+
+This function checks whether a method is energy preserving 
+for a given order s. 
+    Inputs: 
+        A: square matrix
+        b: vector of Butcher array
+        s: integer, the order
+    Output:
+        True/False
+"""
+function is_energy_preserving(A,b,s)
```

Comment:
For consistency with the other functions in this package, it would be nice to change this to
```suggestion
function is_energy_preserving(rk::RungeKuttaMethod, order)
```
Then, the implementation should just create a B-series of the appropriate `order` and pass it to `is_energy_preserving(series)`

Reply:
It might be nice to allow both interfaces.  But yes, we definitely want the one that takes a RungeKuttaMethod.

Also, let's follow the convention of using "p" for order ("s" is usually the # of stages).

Reply:
Implemented.